core/src/main/resources/application.conf

```diff
@@ -0,0 +1,23 @@
+"esloader": {
+  "input": {
+    "maxRecords": 10000
+    "appName": "snowplow-elasticsearch-loader"
+    "buffer": {
+      "byteLimit": 1000000
+      "recordLimit": 500
+      "timeLimit": 500
+    }
+  }
+  "output": {
+    "good": {
+      "client": {
+        "maxTimeout": 10000
+        "maxRetries": 6
+      }
+      "chunk": {
+        "byteLimit": 1000000
+        "recordLimit": 500
+      }
+    }
+  }
```

Comment:
How about adding also
```
  "monitoring": {
    "metrics": {
      "cloudwatch": true
    }
  }
```

This means you can remove some of the `Option`s in your monitoring and metrics configs, because they are guaranteed always defined.  And we are more clear about what we set to be the default behaviour.

Reply:
I think that would be good change. Then, I need to wrap [SnowplowMonitoring](https://github.com/snowplow/snowplow-elasticsearch-loader/blob/6dec51cd00691b80a9fe7706749a9cb8828954f6/core/src/main/scala/com.snowplowanalytics.stream/loader/Config.scala#L175) to `Option` because we don't have any default for that I guess.